src/components/QRCodeModal/index.tsx

```diff
@@ -0,0 +1,84 @@
+import QRCode from 'qrcode.react';
+import React, { useEffect, useRef } from 'react';
+import { FcDownload } from 'react-icons/fc';
+import { IoCloseSharp } from 'react-icons/io5';
+
+import Button from '@/components/Button';
+const RDSIcon = '_next/image?url=%2Frds.png&w=64&q=75';
```

Comment:
This is interesting approach, why not image from public folder?

Reply:
I attempted various methods, such as including the image URL for RDS icons, but encountered a cross-origin error. Using a static image from the public directory didn't work either, as the QR code requires dynamic string data.